python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/defs_flow_tracking.schema.yml

```
@@ -61,7 +61,7 @@ $defs:
                       warning: true
                       removed: false
                       remove_in_version: 6.0.0
-                      new_key: collectors
+                      new_key: flow_tracking.trackers[].exporters[].collectors
```

```suggestion
                      new_key: collectors
```
By keeping only collectors key you can trigger the error when both keys are defined. With the full new key the error will not be raised when define both keys